components/encumbered/minidlna/patches/03-monitor.c.patch

```diff
@@ -0,0 +1,41 @@
+diff -ur minidlna-1.2.1/monitor.c minidlna-1.2.1-mod/monitor.c
```

Comment:
@citrus-it - it seems this patch is more sane, as is_dir() and is_reg() are defined on struct dirent*, not char* and always return -1 on illumos.